### PR TITLE
Author nickname MUST be prefixed by `@`

### PR DESCRIPTION
**Pull request by vjik**
| Q             | A
| ------------- | ---
| Is bugfix?    | ❌
| New feature?  | ✔️
| Breaks BC?    | ❌
| Fixed issues  | -

### CODE REVIEW

**Review by rustamwin — APPROVED** [review]

**Review by terabytesoftw — APPROVED** [review]

File: 021-changelog-upgrade.md [hunk 1]
```diff
@@ -33,7 +33,7 @@ For each version there are number of lines listing the changes. "Bug" refers to
 In the changelog file lines should be ordered as New, Chg, Enh, Bug.
 
 "#42" above is the number of issue or pull request corresponding to the change. "author1" is the GitHub nickname of the
-code author. "author2" is additional author.
+code author. "author2" is additional author. Author nickname MUST be prefixed by `@`.
```
Comment (arogachev) [inline 021-changelog-upgrade.md:36]:
```suggestion
code author. "author2" is additional author. An author's nickname *must* be prefixed with `@`.
```

File: 021-changelog-upgrade.md [hunk 2]
```diff
@@ -15,7 +15,7 @@ Changelog is written for each version released. The file name is `CHANGELOG.md`.
 
 ## 1.0.1 March 23, 2021
 
-- Bug #42: Short description of the change (author1, author2)
+- Bug #42: Short description of the change (@author1, @author2)
```
Comment (arogachev) [inline 021-changelog-upgrade.md:18]:
```suggestion
- Bug #42: Short description of the change (`@author1`, `@author2`)
```

**Review by arogachev — CHANGES_REQUESTED** [review]

File: 021-changelog-upgrade.md (hunk 2 above)
Comment (vjik) [reply 021-changelog-upgrade.md:18]: Not need wrap nicknames as code: https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#mentioning-people-and-teams

File: 021-changelog-upgrade.md (hunk 1 above)
Comment (vjik) [reply 021-changelog-upgrade.md:36]: Keywords "must", "should" and etc. are usually written in uppercase. For example, [PSR documents](https://www.php-fig.org/psr/psr-11/) or [Yii docs](https://github.com/yiisoft/docs/blob/master/000-packages.md#php-versions-support).

**Review by samdark — APPROVED** [review]

**Pull request merged**